validator/python/README.md

```diff
@@ -0,0 +1,41 @@
+# GeoParquet validator - Python
+
+Command-line tool to validate a GeoParquet file. Written in Python. Using [JSON Schema](https://json-schema.org/).
+
+## Installation
+
+```
+pip install wheel
+pip install .
```

Comment:
Why is `pip install wheel` necessary? We're not building any wheels here..?

Reply:
Yep, it's a bit weird. I tested in a fresh venv and found this "error"

```
Building wheels for collected packages: geoparquet-validator
  Building wheel for geoparquet-validator (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/jesus/code/geoparquet/validator/python/venvt/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-2458mz9x/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-2458mz9x/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-qcay0ncr
       cwd: /tmp/pip-req-build-2458mz9x/
  Complete output (6 lines):
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
  
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for geoparquet-validator
  Running setup.py clean for geoparquet-validator
Failed to build geoparquet-validator
Installing collected packages: geoparquet-validator
    Running setup.py install for geoparquet-validator ... done
Successfully installed geoparquet-validator-0.0.1
```

It's true that it can be removed by using no-binary for the package:

```
pip install --no-binary geoparquet_validator .
```